holoviews/plotting/bokeh/plot.py

```diff
@@ -23,6 +23,10 @@
 
 from bokeh.layouts import gridplot
 from bokeh.plotting.helpers import _known_tools as known_tools
+try:
+    from bokeh.themes import built_in_themes
```

Comment:
philippjfr suggested:
```
if bokeh_version < '0.13.0':
    built_in_themes = {}
else:
    from bokeh.themes import built_in_themes
```

But I wasn't sure if could compare whether less than/greater than with a string so try and except seemed more straightforward to me

Reply:
bokeh_version (from plotting/bokeh/util.py) is of type [LooseVersion](http://epydoc.sourceforge.net/stdlib/distutils.version.LooseVersion-class.html), which does support comparisons with strings and should do the right thing.

Reply:
(note that I have no opinion between that and try/except; just pointing out that string comparison is a valid suggestion here.)

Reply:
No objection to the try/except, but @jbednar is correct about string comparisons being supported.

Reply:
Too late, learned something new heh